acceptance-tests/support/steps/input-steps.ts

```diff
@@ -0,0 +1,40 @@
+import {When, Then} from "@cucumber/cucumber";
+import {enterTextIntoTextInput, checkErrorMessageDisplayedForField, clickSubmitButton} from "./shared-functions";
+
+const fields: Record<string, string> = {
```

Comment:
```suggestion
const fields = {
```
Remove the type declaration to create an object literal type and get typechecking when accessing the properties in the steps below.